src/inverseRobot.js

```diff
@@ -8,6 +8,17 @@
 
 function inverseRobot(robot) {
   // write code here
+  const values = Object.values(robot);
+  const result = {};
+
+  for (const key in robot) {
+    if (values.indexOf(robot[key]) !== values.lastIndexOf(robot[key])) {
+      return null;
+    }
```

Comment:
you can simplify it by just checking `result[robot[key]]` - then you won't need `values` variable at all
```suggestion
    if (result[robot[key]]) {
      return null;
    }
```